src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/nuget.config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key = "svcutilTestFeed" value="$repositoryRoot$/artifacts/Packages/Release/Shipping" />
+    <add key = "svcutilTestFeed" value="$repositoryRoot$/artifacts/packages/Release/Shipping" />
     <add key = "nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
is this change necessary?

Reply:
Yes, this is to align with the actual result generated on Linux OS which path name is case-sensitive.